### PR TITLE
Images by file 2.0

### PR DESCRIPTION
**Pull request by ArgaAlan**

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -48,13 +48,13 @@ public void storeMessage(Message message) {
...
     messageEntity.setProperty("longitude", message.getLng());
+    messageEntity.setProperty("image", message.getImageUrl());
     datastore.put(messageEntity);
   }
 
   public void deleteMessageWithID(String messageID) {
-    Query query =
-        new Query("Message")
-            .setFilter(new Query.FilterPredicate("ID", FilterOperator.EQUAL, messageID));
+    Query query = new Query("Message")
```
Comment (acolwell) [inline src/main/java/com/google/codeu/data/Datastore.java]: Please run "mvn com.coveo:fmt-maven-plugin:format" to remove these unrelated formatting changes.

File: src/main/java/com/google/codeu/data/Message.java [hunk 2]
```diff
@@ -1,17 +1,15 @@
 /*
  * Copyright 2019 Google Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
```
Comment (acolwell) [inline src/main/java/com/google/codeu/data/Message.java]: Please restore old license text formatting.

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 3]
```diff
@@ -87,9 +95,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.relaxed());
+    String text = Jsoup.clean(request.getParameter("text"),
+        Whitelist.relaxed().addTags("figure", "oembed").addAttributes("oembed", "url"));
     String textWithMedia = getMediaEmbeddedText(text);
 
+    System.out.println(request.getParameter("text"));
```
Comment (acolwell) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java]: Please remove debug print statements.

**Review by acolwell — CHANGES_REQUESTED** [review]
Looks pretty good. Just a few minor things to fix and I think you can merge this.

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (ArgaAlan) [reply src/main/java/com/google/codeu/data/Datastore.java]: Done

File: src/main/java/com/google/codeu/data/Message.java (hunk 2 above)
Comment (ArgaAlan) [reply src/main/java/com/google/codeu/data/Message.java]: Done

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 3 above)
Comment (ArgaAlan) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java]: Done

**Review by acolwell — APPROVED** [review]
LGTM
